app/prepare_proposal.go

```diff
@@ -59,6 +59,8 @@ func (app *App) PrepareProposal(req abci.RequestPrepareProposal) abci.ResponsePr
 	}
 
 	// erasure the data square which we use to create the data root.
+	// Note: uses the nmt wrapper to construct the tree.
+	// checkout pkg/wrapper/nmt_wrapper.go for more information.
```

Comment:
[I'll do this in a different PR]
oh woah we should definitely add docs to https://github.com/celestiaorg/celestia-app/blob/24626ff8d00854d7e0027abcfdf50de1a2aee6a3/pkg/da/data_availability_header.go#L52

Reply:
Thanks :+1: